docs/products/clickhouse/howto/use-shards-with-distributed-table.rst

```diff
@@ -0,0 +1,61 @@
+Enable reading and writing data across shards in Aiven for ClickHouse®
+======================================================================
+
+If your Aiven for ClickHouse® service uses multiple shards, the data is replicated only between nodes of the same shard. When you read from one node, you see the data from the shard of this node. In this article, you'll learn how to create a `distributed table <https://clickhouse.com/docs/en/engines/table-engines/special/distributed/>`_ on top of your replicated table, which will not only allow you to see data from all the shards but also help spread the data evenly across all the cluster nodes.
+
+Set up a sharded service with a database
+----------------------------------------
+
+.. note::
+
+    For Aiven for ClickHouse® services, shards are created automatically when the services are created. The number of shards that your service gets depends on the plan you select for your service. You can calculate the shards number as follows: *number of shards = number of nodes / 3*.
+
+1. :doc:`Create an Aiven for ClickHouse® service </docs/platform/howto/create_new_service>` with multiple shards.
+2. :ref:`Create database <create-a-clickhouse-database>` *test_db* in your new service.
+
+Create a distributed table
+--------------------------
+
+1. :doc:`Connect to your database </docs/products/clickhouse/howto/connect-to-ch-service>`.
+
+2. Create a table with the `MergeTree engine <https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/mergetree/>`_ as shown for the ``cash_flows`` table in the following example::
+
+    CREATE TABLE test_db.cash_flows
+    ( 
+    EventDate DateTime,
+    SourceAccount UInt64,
+    TargetAccount UInt64,
+    Amount Float64
+    )
+    ENGINE = MergeTree()
+    PARTITION BY toYYYYMM(EventDate)
+    ORDER BY (EventDate, SourceAccount)
+
+.. note::
+
+    ``<table-columns-and-data-types>`` is a placeholder for including all table columns and their data types (see avaialble types and properties in the open-source ClickHouse® documentation: `SQL reference <https://clickhouse.com/docs/en/sql-reference/>`_ and `Data types <https://clickhouse.com/docs/en/sql-reference/data-types/>`_).
+
+3. Create distributed table ``cash-flows_distributed`` with the distributed engine::
+
+    CREATE TABLE test_db.cash_flows_distributed AS test_db.cash_flows
+    ENGINE = Distributed(test_db, test_db, cash_flows, SourceAccount)
+
+Verify your distributed table
+-----------------------------
+
+Check if the distributed table you created is available and if you can use it to access your data from all the shards.
+
+1. Run a read query for the number of table rows::
+
+    SELECT count() FROM test-db.cash-flows_distributed
```

Comment:
There is something wrong with consistency "-" vs "\_" in the names given to table and to db. Seems to me like a typo.
Generally whenever possible for Clickhosue better to use "\_", because "-" won't work with ClickHouse and needs to be surrounded in back quotes "\`" \`like-this\`.